annlite/index.py

```diff
@@ -621,6 +619,31 @@ def snapshot_path(self):
             return paths[0]
         return None
 
+    @property
+    def remote_store_client(self):
+        try:
+            import hubble
+
+            client = hubble.Client(max_retries=None, jsonify=True)
+            client.get_user_info()
+            return client
+        except Exception as ex:
+            logger.error(f'Not login to hubble yet.')
+            raise ex
+
+    def backup(self, target_name: Optional[str] = None):
+        if not target_name:
+            self.dump_index()
+        else:
+            self._backup_index_to_remote(target_name)
+
+    def restore(self, source_name: Optional[str] = None):
+        if not source_name:
+            if self.total_docs > 0:
```

Comment:
I tried to restore the indexer from the local machine and I found that if you require the total_docs > 0, it won't be restored because once a new indexer is initialized, the total_docs will always be 0.

Reply:
OK I know what happens here, `total_docs` comes from rocksdb and it will always be bigger than 0 if we have data there, otherwise, it's an empty indexer.